docs/quick-start.md

```diff
@@ -30,14 +30,14 @@ When you complete the registration, you'll be given a **Client ID** and a **Clie
 ## App File Structure
 All Netlify CMS files are contained in a static `admin` folder, stored at the root of the generated site. Where you store this in the source files depends on your static site generator. Here's the the static file location for a few of the most popular static site generators:
 
-These generators... | store static files in
+These generators ... | store static files in
 --- | ---
 Jekyll, GitBook | `/` (project root)
 Hugo | `/static`
 Hexo, Middleman | `/source`
 Spike | `/views`
 
-If your generator isn't listed here, you can check its documentation, or as a shortcut, look in your project for a `CSS` or `images` folder. They're usually processed as static files, so it's likely you can store your `admin` folder next to those. (When you've found the location, feel free to add it to these docs!).
+If your generator isn't listed here, you can check its documentation, or as a shortcut, look in your project for a `CSS` or `images` folder. They're usually processed as static files, so it's likely you can store your `admin` folder next to those. (When you've found the location, feel free to add it to these docs by filing a pull request!).
```

Comment:
Seems like a nice place to link https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md, though we did just do that earlier in the page.